substrate-query-framework/index-builder/src/entities/SubstrateEventEntity.ts

```diff
@@ -68,9 +65,9 @@ export class SubstrateEventEntity {
     _entity.index = q.indexInBlock;
     _entity.id = SubstrateEventEntity.formatId(_entity.blockNumber, _entity.index);
     _entity.method = q.event_record.event.method || 'NO_METHOD';
-    _entity.name = q.event_name;
-    _entity.phase = q.event_record.phase.toJSON();
     _entity.section = q.event_record.event.section || 'NO_SECTION';
+    _entity.name = `${_entity.section}.${_entity.method}`;
```

Comment:
Event handler name pattern was `section_method` and we do comparison by event name in order to call mapper.

Reply:
Actually it does not (and should not) affect the mappers, as it's just the raw events data. The mapping code itself should responsible for matching the handlers with the entity names.

Reply:
👍